regexes/device/consoles.yml

```diff
@@ -35,10 +35,19 @@ OUYA:
   device: 'console'
   model: 'OUYA'
 
+Sanyo:
+  regex: '.*_SANYO_.*'
+  device: 'console'
+  model: ''
+
 Sega:
-  regex: 'Dreamcast'
+  regex: 'Dreamcast|.*_SEGASATURN_.*'
```

Comment:
```
regex: 'Dreamcast|Aplix_SEGASATURN'
```
greedy modifiers are best avoided if possible (there is only 1 browser under sega saturn, and then it was accessed via a cartridge/disk)